### DISCUSSION TITLE
How do i blur content on a my website

### DISCUSSION THREAD

**Discussion by keffkelvin** [community/community#40089] [Other Feature Feedback, Questions, & Ideas]
Hi, someone kindly help me know how to blur content i would like people to pay to access on my website. My website is created using joomla

**Comment by Aryan-Jadon18** [comment]
You can blur an image on a website by using this CSS code:
 filter: blur(8px);
  -webkit-filter: blur(8px);

**Comment by Preetiraj3697** [comment]
background-color: rgba(0, 0, 0, 0.61);
backdrop-filter: blur(5px);
